besu/src/main/java/org/hyperledger/besu/RunnerBuilder.java

```diff
@@ -790,7 +790,17 @@ public Runner build() {
       LOG.debug("added ethash observer: {}", stratumServer.get());
     }
 
-    sanitizePeers(network, staticNodes)
+    final Stream<EnodeURL> maintainedPeers;
+    if (besuController.getGenesisConfigOptions().isPoa()) {
+      // In a permissioned chain Besu should maintain connections to both static nodes and
+      // bootnodes, which includes retries periodically
+      maintainedPeers =
+          Stream.concat(sanitizePeers(network, staticNodes), sanitizePeers(network, bootnodes));
```

Comment:
would it be more efficient to join the 2 collections and call sanitizePeers only once?

Reply:
suggest a debug log message that this is happening

Reply:
Thanks @macfarla - both comments should now be addressed

Reply:
> I would just make sure also that the logging of static nodes, bootnodes etc at startup still makes sense
I'm hoping that I haven't affected any existing maintained-peer logs. The maintained-peer list also contains any nodes added manually with `admin_addPeer` so it's not purely a static-nodes list today.